src/picongpu/include/plugins/hdf5/restart/LoadSpecies.hpp

```diff
@@ -202,40 +202,9 @@ struct LoadSpecies
 
             const uint32_t cellsInSuperCell = PMacc::math::CT::volume<SuperCellSize>::type::value;
 
-            const uint32_t iterationsForLoad = ceil(float_64(totalNumParticles) / float_64(restartChunkSize));
```

Comment:
Please remove `counterBuffer` and the comment. Some lines above this code (line 192 - 201)